cpp/models/abm/world.h

```diff
@@ -195,6 +195,8 @@ class World
     MigrationParameters m_migration_parameters;
     TripList m_trip_list;
     bool m_use_migration_rules;
+    using migration_rule = LocationType (*)(const Person&, TimePoint, TimeSpan, const MigrationParameters&);
+    std::vector<std::pair<migration_rule, std::vector<LocationType>>> global_migration_rules;
```

Comment:
please use m_* for member variables; see also https://github.com/DLR-SC/memilio/wiki/cpp-coding-guidelines

Reply:
Thanks for your comment. It's done and pushed.